llvmlite/binding/ffi.py

```diff
@@ -151,47 +151,19 @@ def __call__(self, *args, **kwargs):
             return self._cfn(*args, **kwargs)
 
 
-_lib_dir = os.path.dirname(__file__)
-
-if os.name == 'nt':
-    # Append DLL directory to PATH, to allow loading of bundled CRT libraries
```

Comment:
Why / when would CRT libraries be bundled with llvmlite?

Reply:
(note, this is mainly a question for @sklam / @stuartarchibald / @esc, not necessarily a bit of feedback about the PR - I'm finding it hard to imagine a situation where this makes sense)

Reply:
Answering my own question - it was added in https://github.com/numba/llvmlite/commit/1f7133c5 which we no longer seem to do (presumably it was needed for the newer libraries needed by LLVM on Python 2 which would have been linked to much older runtimes)